### ISSUE TITLE
sphinx-quickstart creates non-satisfying html default set-up with Alabaster

### ISSUE THREAD

**Issue by jfbu** [sphinx-doc/sphinx#5066]
I tested 1.7.5 and current master, both consider ``'alabaster'`` to be default HTML theme, but the `conf.py` is ill-configured for them and creates un-navigable documents.

For example, this:

![capture d ecran 2018-06-09 a 15 27 38](https://user-images.githubusercontent.com/2589111/41192121-b0f356b0-6bf9-11e8-983c-1cdc4a1f405e.png)

with an `index.rst` like this

```rst
.. toctree::
   :maxdepth: 2

   foo1
   foo2
```

There is no link to go back to table of contents or to foo1 page.

#### Expected results

Things should work out-of-the box. It does not make sense to make default ``alabaster`` and not provide correct set-up to use it.

### Environment info
- OS: Mac
- Python version: 3.6.5
- Sphinx version: 1.7.5 and master
- Alabaster theme 0.7.10

Relates:

#3830, #4496, #3849, #3987, #4002 and probably others. Edit: #4177

I know this has been issued many times and I did personally at least two commits to fix it but it always end up overridden in some way. So I would really like this to be fixed once and for all.

Update: later investigation in my comments below reminded me #4177 should have fixed this, but there is a problem remaining when theme does not provide sidebars config in theme.conf.

**Comment by jfbu** [comment]
I have raised the issue upstream at Alabaster: https://github.com/bitprophet/alabaster/issues/134

**Comment by jfbu** [comment]
For some reason the conditional test for ``'alabaster'``at  

https://github.com/sphinx-doc/sphinx/blob/b371312887869c52e7be9033e60450a9dc12ca0c/sphinx/builders/html.py#L915-L921

appears to be without effect?

**Comment by jfbu** [comment]
Ah I see, it should have been 'relations.html' .... 

edit: actually no, that's not enough because ``'relations.html'`` also need extra activation in custom.css because alabaster by default has

```
div.relations {
    display: none;
}
```

so this is not the only cause of issue. But adding `_static/custom.css` with 

```
div.relations {
    display: block;
}
```

I do get navigable contents.

MEMO: http://alabaster.readthedocs.io/en/latest/installation.html#installation

**Comment by jfbu** [comment]
@tk0miya  trying to debug this with suitable print() added to sphinx.builder.html.StandaloneHTMLBuilder.add_sidebars() I see the `'alabaster'` branch is never taken and sidebars are of the type

```
['localtoc.html', 'relations.html', 'sourcelink.html', 'searchbox.html']
```

which does not include ``navigation.html``. (nor ``about.html`` which is needed by Alabaster).

**Comment by jfbu** [comment]
Notice that if manually extend Alabaster's 0.7.10's provided theme.conf like this for example:

```
[theme]
inherit = basic
stylesheet = alabaster.css
pygments_style = alabaster.support.Alabaster
sidebars = about.html
```

then the expected behaviour is observed.

So the bug is in case the html theme provides no sidebars config value by itself. Then something makes the conditional checking for alabaster never gets executed.

Besides the latter has a typo which I have signalled above, but that's not root cause, as anyhow additional CSS customization is needed for ``relations.html`` to have any effect.

**Comment by jfbu** [comment]
The ``theme.get_selfconfig('theme', 'sidebars', None)``  leads to execution of ``<Theme object>.base.get_config('theme', 'sidebars', None)`` hence the problem. For some reason ``<Theme Object>.base`` is not None.

**Comment by tk0miya** [comment]
Fixed by #5094 
Thank you for reporting :-)
